hera_qm/ant_metrics.py

```diff
@@ -10,6 +10,32 @@
 from hera_qm import utils
 
 
+ant_metrics_list = {'ant_metrics_meanVij': 'Mean of the absolute value of all '
```

Comment:
Ok. I'm working on the functions, will push a commit soon. I also added a comment to the existing issue #36 (unify metric structures) to make sure we revisit the broad documentation. I'm reluctant to write up a memo right now because I _hope_ the interface gets a lot more simple when we unify things.